pkg/kcp/nuke/reconciler.go

```diff
@@ -40,7 +47,8 @@ func (r *nukeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 func (r *nukeReconciler) newAction() composed.Action {
 	return composed.ComposeActions(
 		"nukeMain",
-		composed.LoadObj,
+		feature.LoadFeatureContextFromObj(&cloudcontrolv1beta1.NfsInstance{}),
```

Comment:
⚠️ but the feature Nuke does not exist, so atm it will be logged with unknown feature I guess, which is better then misleading nfs feature, ideally let's expand the feature enum with Nuke feature and let Nuke implement `featurestypes.FeatureAwareObject`
```suggestion
		feature.LoadFeatureContextFromObj(&cloudcontrolv1beta1.Nuke{}),
```